src/style.css

```diff
@@ -1 +1,29 @@
 /* add styles here */
+.page__body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin: 0 4px 0 0;
```

Comment:
To maintain consistency in your margins and to avoid potential margin collapse issues, it's recommended to add margins to only one side (either top or bottom, but not both). Here you have a right margin, which is fine, but just be mindful of vertical margins.